api/controllers/NetworkController.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
     var setNetworkIdIfModerator = (communityId, networkId, trx) => {
       return Membership.hasModeratorRole(req.session.userId, communityId)
       .then(isModerator => {
-        if (isModerator) {
+        if (isModerator || Admin.isSignedIn(req)) {
```

Comment:
this and the upper portion of the diff are suspiciously similar -- factor out into a function pls